app/content/app.css

```diff
@@ -1,11 +1,47 @@
-body {
-  font-family: Oxygen, Helvetica, Arial, sans-serif; }
-
 h1, h2, h3, h4, h5 {
-font-family: Merriweather;
-font-weight: 300;
-color: #FE5F55; }
+font-family: Oxygen, Helvetica, Arial, sans-serif;
+color: #FE5F55;
+}
+
+.btn-register {
+  background-color: #FE5F55;
+}
+
+.btn-register:hover {
+  background-color: #fc8b85;
+}
 
+.btn-login {
+  background-color: #FE5F55;
+}
+
+.btn-login:hover {
+  background-color: #fc8b85;
+}
+
+.top-bar {
+  background-color: #FE5F55;
+}
+
+.nav-item-text {
+  background-color: #FE5F55;
+  color: white !important;
```

Comment:
We should try to avoid using `!important` as much as we can. `!important` overrides how styles apply so if we want to use `.nav-item-text` later on we won't be able to override these styles easily.

Alternatively, we would try to scope the styles a little further. Something like this should work:
```
.menu .nav-item-text {
  background-color: #FE5F55;
  color: white !important;
}
```